packages/iov-lisk/src/liskconnection.ts

```diff
@@ -100,10 +106,40 @@ export class LiskConnection implements BcpConnection {
       throw new Error("Did not get meta.status: true");
     }
 
+    let blockInfoInterval: any;
+    let lastEventSent: BcpBlockInfo | undefined;
+    const blockInfoProducer = new DefaultValueProducer<BcpBlockInfo>(
+      {
+        state: BcpTransactionState.Pending,
+      },
+      {
+        onStarted: () => {
+          blockInfoInterval = setInterval(async () => {
+            const search = await this.searchTx({ hash: toAscii(transactionId) as TxId, tags: [] });
+            if (search.length > 0) {
+              const confirmedTransaction = search[0];
+              const event: BcpBlockInfo = {
+                state: BcpTransactionState.InBlock,
+                height: confirmedTransaction.height,
+                confirmations: confirmedTransaction.confirmations,
+              };
+
+              if (!equal(event, lastEventSent)) {
+                blockInfoProducer.update(event);
+                lastEventSent = event;
+              }
+            }
+          }, 3000);
```

Comment:
Can you pull this constant (poll frequency) into a named variable at the top of the file (or possibly as an optional argument to the LiskConnection constructor)?